### PR TITLE
Instagram channel registration and UI implementation of IG assets

### PR DESCRIPTION
**Pull request by mmythily**
Primary reviewer: @GPaoloni 

## Description
- Set up and registered custom Instagram Channel
- Added Instagram logo and color in All Tasks and Timeline components
- Added queue status functionality to Contacts waiting widget for incoming instagram messages
- Changes made according to [this](https://marvelapp.com/prototype/6e73aca/screen/85221384) design document. Note a change is made to use solid color instead of gradient for the icon at Timeline component after discussion with Jana

### Checklist
- [x] Corresponding issue has been opened
- [ ] New tests added
- [N/A] Strings are localized

### Verification steps
- Note the four additions/changes to check for Instagram channel UI integration: Instagram contact window box, Instagram icon at All Tasks and Timeline components, instagram color at chat window start. 
Instagram UI changes

### CODE REVIEW

File: plugin-hrm-form/src/utils/setUpComponents.js [hunk 1]
```diff
@@ -451,3 +454,30 @@ export const setupTwitterChatChannel = () => {
 
   Flex.TaskChannels.register(TwitterChatChannel);
 };
+
+export const setupInstagramChatChannel = () => {
+  const icon = <InstagramIcon width="1em" height="1em" color="#fff" />;
```
Comment (GPaoloni) [inline plugin-hrm-form/src/utils/setUpComponents.js:459]: Maybe I prefer `px` here to be consistent with the rest of measures we have in the app, but I can understand if there's any particular reason to go `em`s instead.

**Review by GPaoloni — APPROVED** [review]
Looks very good! 

One thing I just realized is missing is `Instagram` as an option for `Channel` input in the offline contact form. Maybe is easy to include it in this PR?

Also, for everyone in the team, note that for this to properly work we also need to deploy https://github.com/techmatters/hrm/pull/146.

File: plugin-hrm-form/src/utils/setUpComponents.js (hunk 1 above)
Comment (mmythily) [reply plugin-hrm-form/src/utils/setUpComponents.js:459]: Absolutely, I used em out of habit. I agree it should be px to stay consistent.

**Review by stephenhand — APPROVED** [review]
Not the most knowledgeable about this code but LGTM

**Review by murilovmachado — APPROVED** [review]
The code looks great to me!

**Pull request merged**